### PR TITLE
Project structure improvements - moving exported consts to constants.ts

### PR DESCRIPTION
**Pull request by ctw-joao-luis**
**User-Facing Changes**
N/A

**Description**
Moved constants from the types.ts and index.tsx files into dedicated files created specifically for storing and exporting constants. This change improves the project's structure by ensuring better organization, separating concerns, and making the codebase more maintainable.

**Checklist**

- [X] The web version was tested and it is running ok
- [X] The desktop version was tested and it is running ok
- [ ] This change is covered by unit tests
- [X] Files constants.ts, types.ts and *.style.ts have been checked and relevant code snippets have been relocated

### CODE REVIEW

File: packages/suite-base/src/players/constants.ts [hunk 1]
```diff
@@ -29,3 +29,27 @@ export const FIELD_SIZE_BY_PRIMITIVE: Record<string, number> = {
   string: 20, // we don't know the length upfront, assume a fixed length
 };
 export const MAX_NUM_FAST_PROPERTIES = 1020;
+
+// Capabilities that are not shared by all players.
+export const PlayerCapabilities = {
```
Comment (luluiz) [inline packages/suite-base/src/players/constants.ts:34]: Please, rename it using refactoring (usually F2 in vscode)
```suggestion
export const PLAYER_CAPABILITIES = {
```

**Review by luluiz — CHANGES_REQUESTED** [review]

File: packages/suite-base/src/players/UserScriptPlayer/constants.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+// make sure to use import type to avoid bringing in the actual implementations to the bundle
+export const DiagnosticSeverity = {
```
Comment (luluiz) [inline packages/suite-base/src/players/UserScriptPlayer/constants.ts:5]: uppercase and underscore

File: packages/suite-base/src/players/UserScriptPlayer/constants.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  Hint: 1,
+  Info: 2,
+  Warning: 4,
+  Error: 8,
+};
+
+export const Sources = {
```
Comment (luluiz) [inline packages/suite-base/src/players/UserScriptPlayer/constants.ts:12]: uppercase

File: packages/suite-base/src/players/UserScriptPlayer/constants.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  Typescript: "Typescript",
+  DatatypeExtraction: "DatatypeExtraction",
+  InputTopicsChecker: "InputTopicsChecker",
+  OutputTopicChecker: "OutputTopicChecker",
+  Runtime: "Runtime",
+};
+
+export const ErrorCodes = {
```
Comment (luluiz) [inline packages/suite-base/src/players/UserScriptPlayer/constants.ts:20]: uppercase and underscore

**Review by luluiz — CHANGES_REQUESTED** [review]

File: packages/suite-base/src/players/constants.ts (hunk 1 above)
Comment (ctw-joao-luis) [reply packages/suite-base/src/players/constants.ts]: Suggestion implemented.

File: packages/suite-base/src/players/UserScriptPlayer/constants.ts (hunk 2 above)
Comment (ctw-joao-luis) [reply packages/suite-base/src/players/UserScriptPlayer/constants.ts]: Suggestion implemented.

File: packages/suite-base/src/players/UserScriptPlayer/constants.ts (hunk 3 above)
Comment (ctw-joao-luis) [reply packages/suite-base/src/players/UserScriptPlayer/constants.ts]: Suggestion implemented.

File: packages/suite-base/src/players/UserScriptPlayer/constants.ts (hunk 4 above)
Comment (ctw-joao-luis) [reply packages/suite-base/src/players/UserScriptPlayer/constants.ts]: Suggestion implemented.

**Review by luluiz — APPROVED** [review]

**Pull request merged**